samples/language/Array.qs

```diff
@@ -27,7 +27,7 @@ namespace MyQuantumApp {
         let stringArray = ["a", "string", "array"];
 
         // A new array expression creating the array `[0, 0, 0, 0, 0, 0, 0, 0, 0, 0]`
-        let intArray = Repeated(0, 10);
+        let repeatedArray = Repeated(0, 10);
```

Comment:
Callout: We are redefining `intArray` to `Repeated(0,10)`, due to which slicing will results in elements being zeros. Hence, changed `intArray = Repeated(0,10)` to `repeatedArray = Repeated(0,10)`, so that outputs for array slice is what is mentioned in the sample.